extensions/amp-story/0.1/amp-story-auto-ads.js

```diff
@@ -89,11 +99,56 @@ export class AmpStoryAutoAds extends AMP.BaseElement {
 
   /** @override */
   layoutCallback() {
+    this.readConfig_();
     this.schedulePage_();
     return Promise.resolve();
   }
 
 
+  /**
+   * load in config from child <script> element
+   * @private
+   */
+  readConfig_() {
+    const children = this.element.children;
+    dev().assert(children.length >= 1, `<${TAG}>: this tag's first child` +
+        'should be a <script> element');
+
+    const child = children[0];
+    user().assert(
+        isJsonScriptTag(child),
+        `The <${TAG}> config should ` +
+        'be inside a <script> tag with type="application/json"');
+
+    try {
+      this.config_ = parseJson(child.textContent);
```

Comment:
this is unrecoverable failure, just let it throw

Reply:
removed